modules/javafx.controls/src/test/java/test/javafx/scene/control/ComboBoxTest.java

```diff
@@ -1334,6 +1341,160 @@ public void defaultConverterCanHandleIncorrectType_2() {
         sl.dispose();
     }
 
+    @Test public void test_EditorKeyInputsWhenPopupIsShowing() {
```

Comment:
minor nit: naming is inconsistent to the other added test below (testExcludeKeyMappingsForComboBoxEditor) - either use an underscore or not (my personal preference is to not use them at all, following general java naming conventions also in tests .. but definitely not overly important :)

Reply:
Removed _, changed to `testEditorKeyInputsWhenPopupIsShowing`